checker.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+import click
+import logging
+import yaml
+import datetime
+import pytz
+import connectors
+
+
+client = boto3.client('acm')
+
+def notify(config, expirings_certs):
+    for expirings_cert in expirings_certs:
+        title = "{} - Certificate {} is going to expire".format(expirings_cert["Criticality"], expirings_cert["CertName"])
+        message = """
+        {} is going to expire and is still in use by :
+        {}
+        DomainName: {}
+
+        This probably means that the rotation of the service hasn't been done yet. Contact the team to trigger a new deployment.
+        """.format(expirings_cert["CertName"], expirings_cert["InUseBy"],expirings_cert["DomainName"])
+        for notifier_config in config["notifiers"]:
+            if "type" in notifier_config and notifier_config["type"].lower() == "zendesk":
+                try:
+                  connectors.generate_zendesk_ticket(notifier_config, title, message)
+                except Exception as e:
+                    logging.error("Can't create zendesk ticket: {}".format(e))
+
+
+@click.command()
+@click.option("--config_file", default="config.yaml", help="Specify the config to use")
+def main(config_file):
+    logging.basicConfig(
+        format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
+
+    with open(config_file, 'r') as stream:
+        try:
+            config = yaml.load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```

Comment:
Use logging instead of printing
`logging.error("Unable to parse file {} : {}".format(config_file, exc))`